fabric-loot-api-v2/src/main/java/net/fabricmc/fabric/api/loot/v2/LootTableEvents.java

```diff
@@ -66,9 +66,9 @@ private LootTableEvents() {
 	 *
 	 * <pre>
 	 * {@code
-	 * LootTableEvents.MODIFY.register((resourceManager, lootManager, id, tableBuilder, source) -> {
+	 * LootTableEvents.MODIFY.register((key, tableBuilder, source) -> {
 	 *     // If the loot table is for the cobblestone block and it is not overridden by a user:
-	 *     if (Blocks.COBBLESTONE.getLootTableId().equals(id) && source.isBuiltin()) {
+	 *     if (Blocks.COBBLESTONE.getLootTableId().equals(key) && source.isBuiltin()) {
```

Comment:
Is the equals check still correct?

Reply:
Technically works, but identity is the convention.

Reply:
Yarn needs update :)